CMakeLists.txt

```diff
@@ -485,11 +496,12 @@ set_target_properties(iio PROPERTIES
 	VERSION ${VERSION}
 	SOVERSION ${LIBIIO_VERSION_MAJOR}
 	FRAMEWORK ${OSX_FRAMEWORK}
-	PUBLIC_HEADER ${LIBIIO_HEADERS}
-	C_STANDARD 99
+        PUBLIC_HEADER "${LIBIIO_HEADERS}"
+        C_STANDARD 99
 	C_STANDARD_REQUIRED ON
 	C_EXTENSIONS OFF
 )
+
```

Comment:
And please unstage this. Although it looks better, we prefer to have the commits as minimal as possible and not carry unrelated changes.